src/network/networkd-sysctl.c

```diff
@@ -40,10 +41,71 @@ static void manager_set_ip_forwarding(Manager *manager, int family) {
                                   enable_disable(t), af_to_ipv4_ipv6(family));
 }
 
+static int sysctl_inotify_callback(sd_event_source *source, const struct inotify_event *event, void *userdata) {
+        const char *desc = NULL, *value;
+        Manager *m = userdata;
+        char *path, *buf;
+        size_t len;
+
+        if (sd_event_source_get_description(source, &desc) < 0)
```

Comment:
uh, the description is supposed to be for debugging purposes. please don#t overload it with actual meaning.

Reply:
Is there another way to retrieve the path of the watched directory?